deploy/srep-1000-ethtool-exporter/08-Secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+data:
+  config.yaml: ImF1dGhvcml6YXRpb24iOgogICJzdGF0aWMiOgogIC0gInBhdGgiOiAiL21ldHJpY3MiCiAgICAicmVzb3VyY2VSZXF1ZXN0IjogZmFsc2UKICAgICJ1c2VyIjoKICAgICAgIm5hbWUiOiAic3lzdGVtOnNlcnZpY2VhY2NvdW50Om9wZW5zaGlmdC1tb25pdG9yaW5nOnByb21ldGhldXMtazhzIgogICAgInZlcmIiOiAiZ2V0Ig==
```

Comment:
Does this have to be a secret?

Reply:
Likely no, but since this is for the RBAC perms it might be preferred? This is borrowed directly from CMO in order to provide access specifically to the openshift-monitoring/prometheus-k8s serviceAccount/token.

Reply:
Yeah I decrypted the value and it's just yaml config with no secret in it. But probably fine if that is what is expected